conf/modules.config

```diff
@@ -121,7 +121,7 @@ process {
     }
 
     withName: HISAT2_BUILD {
-        ext.when = { params.module.split(',').contains('differential_expression') || params.tool.split(',').contains('ciriquant') }
+        ext.when = { params.tool.split(',').contains('ciriquant') }
```

Comment:
It's usually more helpful to put this kind of logic in the workflow code so people understand conditionally executed. Out here it's hidden and can make things harder to debug ( generally ).

Reply:
I agree - and since I took over the pipeline I have already removed a lot of these, but there are still a few left. Will change this in a future PR.